### PR TITLE
Fix snpeff data managers (older builds are missing python)

### PR DESCRIPTION
**Pull request by mvdbeek**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<tool id="data_manager_snpeff_databases" name="SnpEff Databases" version="4.3r" tool_type="manage_data">
+<tool id="data_manager_snpeff_databases" name="SnpEff Databases" version="4.3r" tool_type="manage_data" profile="18.09">
```
Comment (wm75) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml]: `version="4.3t"` ?

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_download.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<tool id="data_manager_snpeff_download" name="SnpEff Download" version="4.3r" tool_type="manage_data">
+<tool id="data_manager_snpeff_download" name="SnpEff Download" version="4.3r" tool_type="manage_data" profile="18.09">
```
Comment (wm75) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_download.xml]: `version="4.3t"` ?

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_download.xml [hunk 3]
```diff
@@ -19,7 +19,7 @@ python '$__tool_directory__/data_manager_snpEff_download.py'
     </outputs>
     <tests>
         <test>
-            <param name="genome_version" value="GRCh38.76"/>
+            <param name="genome_version" value="GRCh38.86"/>
```
Comment (wm75) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_download.xml]: requires `has_text text="GRCh38.86"` below

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml (hunk 1 above)
Comment (mvdbeek) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_databases.xml]:
```suggestion
<tool id="data_manager_snpeff_databases" name="SnpEff Databases" version="4.3t" tool_type="manage_data" profile="18.09">
```

File: data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_download.xml (hunk 2 above)
Comment (mvdbeek) [inline data_managers/data_manager_snpeff/data_manager/data_manager_snpEff_download.xml]:
```suggestion
<tool id="data_manager_snpeff_download" name="SnpEff Download" version="4.3.1t" tool_type="manage_data" profile="18.09">
```

**Comment by bgruening** [comment]
@wm75 ready?

**Pull request merged**